### PR TITLE
move common from hiera data values to init.pp

### PR DESCRIPTION
**Pull request by Dan33l**
#### Pull Request (PR) description

For an easier reading accordingly with https://voxpupuli.org/docs/reviewing_pr/
> Static data that is equal across every supported operating system can stay in the init.pp, it doesn’t need to be moved to a common.yaml

### CODE REVIEW

File: hiera.yaml [hunk 1]
```diff
@@ -15,5 +15,8 @@ hierarchy:
   - name: 'Distribution Name'
     path: '%{facts.os.name}.yaml'
 
+  - name: 'Operating System Family'
+    path: '%{facts.os.family}.yaml'
```
Comment (bastelfreak) [inline hiera.yaml]: Do we still need this?

File: manifests/init.pp [hunk 2]
```diff
@@ -26,70 +26,39 @@
 #   }
 #
 # @param manage_service Disable/Enable the management of the ferm daemon
-#   Default value: false
```
Comment (bastelfreak) [inline manifests/init.pp]: It would be nice if we could keep the allowed value?

File: hiera.yaml (hunk 1 above)
Comment (Dan33l) [reply hiera.yaml]: In one hand, module supports CentOS, RedHat and Fedora.
In second hand, before this PR, common in hiera data handle values for the RedHat family OSes.
And finally, in this PR, we remove the common from hiera data in module.

So, now, yes it is needed to handle values for RedHat family OSes.

File: manifests/init.pp (hunk 2 above)
Comment (Dan33l) [reply manifests/init.pp]: The data type defined the allowed values.
And puppet-strings generates doc by giving the data type of a parameter and so the allow values.

I do not understand the interest to manually maintain what is generated by puppet-strings. By this way we have to maintain two informations that can be not synchronized. 

One exception is when the default value is put into hiera data of the module. In this case puppet-strings does not  include the values contained in hiera data.

**Pull request merged**